test/pages/blog/index.spec.js

```diff
@@ -35,11 +35,33 @@ describe('Conference page', () => {
                     return []
                   },
                 }
+              }
+            }
+          },
+          where: () => {
+            return {
+              only: () => {
+                return {
+                  sortBy: () => {
+                    return {
+                      fetch: () => {
+                        return []
```

Comment:
![image](https://user-images.githubusercontent.com/8480203/149794666-019c1b25-b731-4433-b986-30342f4641fb.png)

Reply:
Nothing to do here. Ich konnte nur nicht widerstehen :D